packages/mui-material/scripts/testModuleAugmentation.js

```diff
@@ -1,7 +1,7 @@
 const childProcess = require('child_process');
 const path = require('path');
 const { promisify } = require('util');
-const { chunk } = require('lodash');
+const chunk = require('es-toolkit/compat/chunk');
```

Comment:
Do we need compat here?

```suggestion
const { chunk } = require('es-toolkit/array');
```